src/modules/translation/languages/es.json

```diff
@@ -794,6 +794,9 @@
     "authorizations_title": "Autorizaciones Activas"
   },
   "transaction": {
+    "approved": "aprobadó",
+    "not_approved": "desaprobó",
+    "approve_token": "{action} al contrato de {contract} para que opere tokens de {token} por tí.",
```

Comment:
```suggestion
    "approve_token": "{action} al contrato {contract} para que opere {token} por tí.",
```

Reply:
This suggestion comes from:
![Screen Shot 2021-08-18 at 11 33 46](https://user-images.githubusercontent.com/1120791/129919304-d248f40d-fc61-45f9-8fc0-95b866f4d1bd.png)